### PR TITLE
Issue/1674 add color to merchant feed

### PR DESCRIPTION
**Pull request by eutopian**
Added color attribute (:item_color) to product feed

### CODE REVIEW

File: lib/spree_google_base/engine.rb [hunk 1]
```diff
@@ -19,7 +19,8 @@ class Engine < Rails::Engine
         ['g:condition', 'google_base_condition'],
         ['g:availability', 'google_base_availability'],
         ['g:google_product_category', 'google_base_product_category'],
-        ['g:item_group_id', 'google_base_item_group_id']
+        ['g:item_group_id', 'google_base_item_group_id'],
+        ['g:item_color', 'google_base_item_color']
```
Comment (sherryson) [inline lib/spree_google_base/engine.rb]: i was looking at the documentation (https://support.google.com/merchants/answer/6324487) and it seems like the attribute should be called `g:color`

File: app/models/spree/variant_decorator.rb [hunk 2]
```diff
@@ -55,6 +55,12 @@ def google_base_item_group_id
       product.slug
     end
 
+    def google_base_item_color 
+      if option_values
+        option_values.first.name if option_values.first
```
Comment (sherryson) [inline app/models/spree/variant_decorator.rb]: i think you should be able to condense this, if a variant has `option_values` there will always be at least a first, right?

so maybe something like `return option_values.first.name if option_values` ?

**Review by sherryson — CHANGES_REQUESTED** [review]
@eutopian just left a couple comments, looks great! just a few things to consider

**Comment by bryanmtl** [comment]
Congrats on your first PR @eutopian !

File: lib/spree_google_base/engine.rb (hunk 1 above)
Comment (eutopian) [reply lib/spree_google_base/engine.rb]: good call!

File: app/models/spree/variant_decorator.rb (hunk 2 above)
Comment (eutopian) [reply app/models/spree/variant_decorator.rb]: I tried that.. but it still got the nil class error... I guess some of them don't have a first if it's an empty array? Not sure why in the db there would be but in dev it's calling no method error for option_values.first.name

**Review by sherryson — APPROVED** [review]
looks great!!

**Pull request merged**